controls/radcolorpicker/styling-and-appearance/appearance.md

```diff
@@ -12,7 +12,7 @@ position: 1
 
 You can customize the appearance of a __RadColorPicker__ by choosing which of the parts to be visible.        
 
-##### 1. You can set the __NoColorVisibility__ property to __Visibility.Visible__ to view the __NoColorField__, otherwise you should set it to false:            
+__1. You can set the NoColorVisibility property to Visibility.Visible to view the NoColorField, otherwise you should set it to false__            

```

Comment:
The bolding here should be done inside the list's content. Example:
<pre>1. __You can set the NoColorVisibility property to Visibility.Visible to view the NoColorField, otherwise you should set it to false__  </pre>

Same for the other list items in this article.